lib/generators/foxinator/setup/templates/cms/app/controllers/concerns/localizify.rb

```diff
@@ -79,8 +79,8 @@ def accept_locale
     if request.env['HTTP_ACCEPT_LANGUAGE'].present?
       parsed_locale = request.env['HTTP_ACCEPT_LANGUAGE'].gsub(/\s+/, '').split(',').first
       parsed_locale = parsed_locale.scan(/^[a-z]{2}-[a-z]{2}|[a-z]{2}/i).first if parsed_locale
-      parsed_locale = parsed_locale.split('-').last.try(:downcase).try(:to_sym) if parsed_locale
-      
+      parsed_locale = parsed_locale.split('-').first.try(:downcase).try(:to_sym) if parsed_locale
```

Comment:
The given example shows a locale with an underscore, but the code is splitting by dash. maybe it's better to split by both. Like:
```ruby
parsed_locale.split(Regexp.new('-|_')).first
```